lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -61,6 +61,7 @@
 #include "clang/Driver/Driver.h"
 #include "llvm/ADT/ArrayRef.h"
 #include "llvm/ADT/StringRef.h"
+#include "llvm/ADT/StringSet.h"
```

Comment:
@adrian-prantl I think I need some help here.

I get the following error when compiling:

```
path/to/swift_workspace/llvm-project/lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp:1476:9: error: no type named 'StringSet' in namespace 'llvm'; did you mean 'StringRef'?
  llvm::StringSet unique_flags;
  ~~~~~~^~~~~~~~~
        StringRef
```

I've triple-checked everything, tried to replace this import with invalid one to make sure I'm editing the right file, cleaned build folder and now I've run out of ideas :(

Reply:
StringSet is relatively new. Does it actually exist on the branch you are compiling on?

Reply:
Otherwise you can use `StringMap<bool>` on master and `StringSet` on master-next and master-rebranch

Reply:
Ah: http://llvm.org/doxygen/StringSet_8h_source.html
I think it's `StringSet<>`.

Reply:
Thanks! This was the issue. Didn't know that when all template args are defaulted you still need to use template marks :(

Now I can confirm that it's compiling, is it possible to run address sanitizer on this branch?

Reply:
There is no way to run the Asan Jenkins job on a PR, but you can run it locally with the `--enable-asan` (and `--enable-ubsan`) build-script options.

Reply:
Thanks! I've run `./swift/utils/build-script -r --lldb --enable-asan --enable-ubsan` locally and it seems to finish successfully